src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -708,6 +708,7 @@ export class SuggestWidget implements IDisposable {
 		this._pendingShowDetails.value = dom.runAtThisOrScheduleAtNextAnimationFrame(() => {
 			this._pendingShowDetails.clear();
 			this._details.show();
+			this._list.ariaDescribedBy = this._details.getId();
```

Comment:
`_details.getId()` is unique only per editor instance so theoretically we have two different `ariaDescribedBy` attributes using the same id (of different dom nodes). Tho, practically this shouldn't happen because suggest needs focus but we could add an instance id to the details widget

Reply:
I don't see any direct references to `suggest.details` so it seems safe to update SuggestWidgetDetails to generate a unique id on construction. Is there a service which will provide a globally unique id? I didn't see a common pattern for this.

Reply:
You can use `generateUuid` or the concat the editor id for which the suggest details are being build, like use `ICodeEditor#getId` inside `SuggestDetails#getId`